fontforge/python.c

```diff
@@ -20092,227 +20092,209 @@ static struct flaglist sfnt_name_str_ids[] = {
+    { "Italian (Switzerland)", 0x810},
     { "Japanese", 0x411},
     { "Kannada", 0x44b},
-    { "Kanuri", 0x471},
-    { "Kashmiri (India)", 0x860},
     { "Kazakh", 0x43f},
     { "Khmer", 0x453},
-    { "Kirghiz", 0x440},
+    { "K'iche", 0x486},
+    { "Kinyarwanda", 0x487},
+    { "Kiswahili", 0x441},
     { "Konkani", 0x457},
     { "Korean", 0x412},
-    { "Korean (Johab)", 0x812},
+    { "Kyrgyz", 0x440},
     { "Lao", 0x454},
     { "Latvian", 0x426},
-    { "Latin", 0x476},
     { "Lithuanian", 0x427},
-    { "Lithuanian (Classic)", 0x827},
-    { "Macedonian", 0x42f},
-    { "Malay", 0x43e},
-    { "Malay (Brunei)", 0x83e},
+    { "Lower", 0x82e},
+    { "Luxembourgish", 0x46e},
+    { "Macedonian (FYROM)", 0x42f},
+    { "Malay (Brunei Darussalam)", 0x83e},
+    { "Malay (Malaysia)", 0x43e},
     { "Malayalam", 0x44c},
     { "Maltese", 0x43a},
-    { "Manipuri", 0x458},
     { "Maori", 0x481},
+    { "Mapudungun", 0x47a},
     { "Marathi", 0x44e},
+    { "Mohawk", 0x47c},
     { "Mongolian (Cyrillic)", 0x450},
-    { "Mongolian (Mongolian)", 0x850},
+    { "Mongolian (Traditional)", 0x850},
     { "Nepali", 0x461},
-    { "Nepali (India)", 0x861},
     { "Norwegian (Bokmal)", 0x414},
     { "Norwegian (Nynorsk)", 0x814},
-    { "Oriya", 0x448},
-    { "Oromo", 0x472},
-    { "Papiamentu", 0x479},
+    { "Occitan", 0x482},
+    { "Odia (formerly Oriya)", 0x448},
     { "Pashto", 0x463},
     { "Polish", 0x415},
-    { "Portuguese (Portugal)", 0x416},
-    { "Portuguese (Brasil)", 0x816},
-    { "Punjabi (India)", 0x446},
-    { "Punjabi (Pakistan)", 0x846},
-    { "Quecha (Bolivia)", 0x46b},
-    { "Quecha (Ecuador)", 0x86b},
-    { "Quecha (Peru)", 0xc6b},
-    { "Rhaeto-Romanic", 0x417},
+    { "Portuguese (Brazil)", 0x416},
+    { "Portuguese (Portugal)", 0x816},
+    { "Punjabi", 0x446},
+    { "Quechua (Bolivia)", 0x46b},
+    { "Quechua (Ecuador)", 0x86b},
+    { "Quechua (Peru)", 0xc6b},
     { "Romanian", 0x418},
-    { "Romanian (Moldova)", 0x818},
+    { "Romansh", 0x417},
     { "Russian", 0x419},
-    { "Russian (Moldova)", 0x819},
-    { "Sami (Lappish)", 0x43b},
-    { "Sanskrit", 0x43b},
-    { "Sepedi", 0x46c},
-    { "Serbian (Cyrillic)", 0xc1a},
-    { "Serbian (Latin)", 0x81a},
-    { "Sindhi India", 0x459},
-    { "Sindhi Pakistan", 0x859},
-    { "Sinhalese", 0x45b},
+    { "Sami (Inari)", 0x243b},
+    { "Sami (Lule) (Norway)", 0x103b},
+    { "Sami (Lule) (Sweden)", 0x143b},
+    { "Sami (Northern) (Finland)", 0xc3b},
+    { "Sami (Northern) (Norway)", 0x43b},
+    { "Sami (Northern) (Sweden)", 0x83b},
+    { "Sami (Skolt)", 0x203b},
+    { "Sami (Southern) (Norway)", 0x183b},
+    { "Sami (Southern) (Sweden)", 0x1c3b},
+    { "Sanskrit", 0x44f},
+    { "Serbian (Cyrillic) (Bosnia and Herzegovina)", 0x1c1a},
+    { "Serbian (Cyrillic) (Serbia)", 0xc1a},
+    { "Serbian (Latin) (Bosnia and Herzegovina)", 0x181a},
+    { "Serbian (Latin) (Serbia)", 0x81a},
+    { "Sesotho", 0x46c},
```

Comment:
The standard calls this “Sesotho sa Leboa”, i.e. Northern Sotho, not Sotho in general.